operator/tests/securedcluster/basic-sc/40-assert.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: central-prometheus-k8s
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: central-prometheus-k8s
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: rhacs-central-auth-reader
+  namespace: kube-system
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
```

Comment:
Should those resources also be included in the central helm chart ?

Reply:
They are https://github.com/stackrox/stackrox/blob/7878c65e42f41ec2de4dee5747c671fd35f5f429/image/templates/helm/stackrox-central/templates/99-openshift-monitoring.yaml#L5-L8
https://github.com/stackrox/stackrox/blob/7878c65e42f41ec2de4dee5747c671fd35f5f429/image/templates/helm/stackrox-central/templates/01-central-10-networkpolicy.yaml#L23-L26
https://github.com/stackrox/stackrox/blob/7878c65e42f41ec2de4dee5747c671fd35f5f429/image/templates/helm/stackrox-central/templates/99-openshift-monitoring.yaml#L77-L81